test/test_scenario_2.py

```diff
@@ -0,0 +1,67 @@
+# Unit and other tests
```

Comment:
No need for this comment. We know it's a test file because it's in the test directory and the name starts with test.